astropy/cosmology/io/latex.py

```diff
@@ -0,0 +1,72 @@
+import astropy.units as u
+from astropy.cosmology.connect import readwrite_registry
+from astropy.cosmology.core import Cosmology
+from astropy.cosmology.parameter import Parameter
+from astropy.table import QTable
+
+from .table import to_table
+
+_FORMAT_TABLE = {
+    "H0": "$$H_0 \r{[\\mathrm{km\\,s^{-1}\\,Mpc^{-1}}]}$$",
+    "Om0": "$$\\Omega_{m,0}$$",
+    "Ode0": "$$\\Omega_{\\Lambda,0}$$",
+    "Tcmb0": "$$T_{0} \r{[\\mathrm{K}]}$$",
+    "Neff": "$$N_{eff}$$",
+    "m_nu": "$$m_{nu} \r{[\\mathrm{eV}]}$$",
+    "Ob0": "$$\\Omega_{b,0}$$",
+    "w0": "$$w_{0}$$",
+    "wa": "$$w_{a}$$",
+    "wz": "$$w_{z}$$",
+    "wp": "$$w_{p}$$",
+    "zp": "$$z_{p}$$",
+}
```

Comment:
Sorry about the mess I have made with all my commits I was trying to squash all commits into one guess I didn’t do it right.
@nstarman Based on the given _FORMAT_TABLE the column headers in the output should match the LaTeX code in the _FORMAT_TABLE.  However, this is not the case and I can't seem to understand why?
For example the output produced by the code below
```python
from astropy.cosmology import Planck18
Planck18.write("example_cosmology.tex", overwrite=True, format="latex")
```
Should be
Output:
```latex
\begin{table}
\begin{tabular}{cccccccc}
cosmology & name & $$H_0{[\mathrm{km\,s^{-1}\,Mpc^{-1}}]}$$ & $$\\Omega_{m,0}$$ & $$T_{0}{[\mathrm{K}]}$$ & $$N_{eff}$$ & $$m_{nu}{[\mathrm{eV}]}$$ & $$\\Omega_{b,0}$$ \\
FlatLambdaCDM & Planck18 & 67.66 & 0.30966 & 2.7255 & 3.046 & 0.0 .. 0.06 & 0.04897 \\
\end{tabular}
\end{table}
```

Reply:
The issue is that `O...` do not have units, so are tripping

``if not isinstance(param, Parameter) or param.unit in (None, u.one):``

Reply:
I think the if-check can be skipped.